js/huobi.js

```diff
@@ -5216,15 +5216,29 @@ module.exports = class huobi extends Exchange {
             //     }
             //
         }
-        const request = {
-            'contract_code': market['id'],
-        };
+        const request = {};
+        if (marketType === 'future') {
+            request['symbol'] = market['baseId'];
+        } else {
+            request['contract_code'] = market['id'];
+        }
         const response = await this[method] (this.extend (request, query));
         const data = this.safeValue (response, 'data');
         const account = this.safeValue (data, 0);
         const omitted = this.omit (account, [ 'positions' ]);
         const positions = this.safeValue (account, 'positions');
-        const position = this.safeValue (positions, 0);
+        let position = undefined;
+        if (marketType === 'future') {
```

Comment:
Since we cannot specify the contract code (only the base currency) every future with that symbol will be eligible (let's say ETH this week, ETH this quarter, etc...) I think we should filter them out, but not 100% this is the best approach since for futures almost looks like an "emulation"